src/Sarif/Writers/LoggingOptions.cs

```diff
@@ -13,15 +13,15 @@ public enum LoggingOptions
         // Indent persisted JSON for easy file viewing.
         PrettyPrint = 0x1,
 
-        // Persist verbose information to log file, such as informational messages.
-        Verbose = 0x2,
+        // Suppress all non-errors to the console and results file
+        Quiet = 0x2,
 
         // Overwrite previous version of log file, if it exists.
         OverwriteExistingOutputFile = 0x4,
 
         // Omit redundant properties, producing a smaller but non-human-readable log.
         Optimize = 0x8,
 
-        All = PrettyPrint | Verbose | OverwriteExistingOutputFile | Optimize
+        All = PrettyPrint | Quiet | OverwriteExistingOutputFile | Optimize
```

Comment:
>Quiet [](start = 28, length = 5)

Ultimately, I think quiet goes away here because it is restricted to the console only. This type could be named LogFilePersistenceOptions.

Reply:
Done.

---
In reply to: [559739880](https://github.com/microsoft/sarif-sdk/pull/2246#discussion_r559739880) [](ancestors = 559739880)